scripts/jira/cloner.py

```diff
@@ -277,48 +317,48 @@ def query_build(issue, user):
 
     args = parser.parse_args()
 
-    connection = jira.JIRA(
+    conn = jira.JIRA(
         server=JIRA_SERVER,
         token_auth=args.token)
 
     jql_query = query_build(args.issue, args.user)
     print(f"JQL Query: '{jql_query}'")
 
     try:
-        query = connection.search_issues(jql_str=jql_query, maxResults=9999)
+        query = conn.search_issues(jql_str=jql_query, maxResults=9999)
         print(f"Scanning {len(query)} issues")
     except Exception as e:
         print(f"Unable to retrieve issues: {e}")
-        exit(1)
+        sys.exit(1)
 
-    actions = []
+    actions_list = []
```

Comment:
[C0103](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/invalid-name.html) is a very broad check for all types of naming conventions; I think we should keep it.

Reply:
The main reason for this change is because `actions` and `query` are conflicting with the local variables defined in other functions. We need to either change in `main` function or in all other functions where it has been used.